metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala

```diff
@@ -1,38 +1,32 @@
 package scala.meta.internal.pantsbuild
 
 import bloop.config.{Config => C}
-import java.nio.file.Paths
-import java.nio.file.Files
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
```

Comment:
Can you please configure IntelliJ to 

* sort everything alphabetically
* don't group prefix with `import a.{b,c}`

Reply:
Done.